docs/docs/20-quickstart.md

```diff
@@ -146,11 +146,15 @@ At the end of this process:
 * The Argo CD dashboard will be accessible at [localhost:31443](https://localhost:31443).
 
   The username and password are both `admin`.
+  
+  ![Argo-dashboard-screenshot](../static/img/argo-dashboard-1.png)
```

Comment:
I don't feel like this screenshot is contributing much. It's basically a blank page, so I'm not sure what purpose it serves other than to back up the claim that something is indeed running as localhost:31443.

I would consider dropping this one.